packages/config/src/validation.ts

```diff
@@ -119,20 +119,88 @@ export const isValidBrowserList = (_key: string, browsers: any): ErrResult | tru
   return true
 }
 
+const isValidExperimentalRetryOptionsConfig = (options: any, strategy: 'detect-flake-but-always-fail' | 'detect-flake-and-pass-on-threshold'): boolean => {
+  if (options != null) {
```

Comment:
Could flip this, return true early, and reduce the nesting here. Then just need an extra `return true` for any cases that fall through.

```js
if (options == null) return true

// retries must be an integer of 1 or greater
const isValidMaxRetries = _.isInteger(options.maxRetries) && options.maxRetries > 0

// ... snip ...

return true
```